Healthy/Classes/Modules/CreateAccount/CreateAccountViewModel.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+
+// MARK: CreateAccountViewModel
+final class CreateAccountViewModel {
+  private var username: String = ""
+  private var email: String = ""
+  private var password: String = ""
+  private var confirmPassword: String = ""
+  private var isChecked: Bool  = false
+  private var onButtonEnabled: (Bool) -> Void = { _ in }
+
+  // MARK: Error Messages
+  var namePrompt: String {
+    if !username.isEmpty && username.isValidName {
+      return ""
+    } else {
+      return "Is not valid name."
+    }
+  }
+
+  var emailPrompt: String {
+    if !email.isEmpty && HealthyEmailValidator().isValid(true) {
+      return ""
+    } else {
+      return "enter a valid email address"
+    }
+  }
+
+  var passwordPrompt: String {
+    if !password.isEmpty && HealthyPassValidator().isValid(true) {
+      return ""
+    } else {
+      return "must be between 8 and 15 characters"
+    }
+  }
+
+  var confirmPWPrompt: String {
+    if !confirmPassword.isEmpty || confirmPassword == password {
+      return ""
+    } else {
+      return "password fields do not match"
+    }
+  }
```

Comment:
Dead code? If not, please move it to a separate private extension.

Reply:
unused now